partner_compassion/views/partner_compassion_view.xml

```diff
@@ -197,10 +189,10 @@
         </field>
     </record>
 
-    <record id="view_partner_form_compassion_contact" model="ir.ui.view">
-    <field name="name">res.partner.compassion.linked.contact.form</field>
+    <record id="view_partner_tree_compassion_contact" model="ir.ui.view">
+    <field name="name">res.partner.tree.compassion</field>
     <field name="model">res.partner</field>
-    <field name="type">form</field>
+    <field name="type">tree</field>
```

Comment:
This change is producing your bug in Travis. It is probably due to a wrong merge during rebase.